### ISSUE TITLE
[Bug][iOS]: Mini player says "Nothing Playing" when something is playing

### ISSUE THREAD

**Issue by gdavis** [gdavis/Mimir-Feedback#134]
![Original-0](https://user-images.githubusercontent.com/110691/172900545-52c8cbe7-9c7d-46c5-8d55-67d54514dc89.jpg)

```json
{
	"deviceMetadata": {
		"batteryPercentage": 74,
		"appUptimeMillis": "1654738594000",
		"connectionStatus": "WiFi",
		"osVersion": "15.5",
		"availableDiskBytes": "2548133888",
		"timezone": "America/Indiana/Indianapolis",
		"architecture": "arm64e",
		"networkType": "LTE",
		"deviceModel": "iPhone13,2"
	},
	"appInfo": {
		"buildNumber": "90",
		"appType": {
			"type": "app",
			"encodingVersion": "2.0",
			"v2AppType": {
				"productType": "app"
			}
		},
		"versionString": "1.0.0",
		"devicePlatform": "iOS",
		"appPlatform": "iOS"
	},
	"identifier": {
		"appVersion": "1.0.0",
		"uploadPlatformSDKIdentifier": "com.apple.platform.iphoneos",
		"id": "AAgjQskGCYobYORc9JvuWMQ",
		"appBuild": "90",
		"bundleIdentifier": "com.grantdavisinteractive.Mimir",
		"adamIdentifier": "1508343025",
		"kind": "screenshot",
		"appType": {
			"type": "app",
			"encodingVersion": "2.0",
			"v2AppType": {
				"productType": "app"
			}
		},
		"platformSDKIdentifier": "com.apple.platform.iphoneos"
	},
	"kind": "screenshot",
	"comment": "Says “nothing playing” but something is playing. ",
	"timestamp": "2022-06-09T01:36:56Z",
	"imageCount": 1
}
```

**Comment by gdavis** [comment]
This seems to happen when syncing with iCloud. Updates made from to the playlist don't seem to update objects listening?

**Comment by gdavis** [comment]
A test fix for this has been done in gdavis/Mimir@6df48b1a507649e7bc41c1ef8a7607e52e650146

**Comment by gdavis** [comment]
Closing for testing in the next version. Will reopen if it comes up again .
